src/components/App/index.tsx

```diff
@@ -7,6 +8,13 @@ import MapView from '../MapView';
 import NotFound from '../404Page';
 import muiTheme from '../../muiTheme';
 
+if (process.env.NODE_ENV && process.env.NODE_ENV !== 'development') {
+  Sentry.init({
+    dsn:
+      'https://a3cc5f470dac4b50b288851a0390f686@o213971.ingest.sentry.io/5242966',

```

Comment:
This probably shouldn't be hardcoded. It's not a secret _strictly_ speaking (in the cybersecurity sense), since it's included in client side code in a "public" app, but it's probably better not to leak it, and given that each deployment is basically autonomous (and quite likely based on slightly different codebases), we might well want to set up a different Sentry account for each country deployment.

I'd pull this from an env variable (_maybe_ with a fall-back value to this key if not found just to make deployments easier, although I think it'd be fine to just throw an error in that case too)